pkg/config/config_test.go

```diff
@@ -104,25 +105,8 @@ func TestAddFlags(t *testing.T) {
 	// RPC flags
 	assertFlagValue(t, flags, FlagRPCAddress, DefaultConfig().RPC.Address)
 
-	// Count the number of flags we're explicitly checking
```

Comment:
this is deleted because using MarkFlagsMutuallyExclusive makes this no more accurate.